src/Action.hs

```diff
@@ -119,7 +120,7 @@ instance ThrowError500 ActionExcept where
 
 class ActionError m => ActionUserHandler m where
     -- | Make the user logged in
-    login  :: User -> m ()
+    login  :: AUID User -> m ()
```

Comment:
The point of taking a user what to somewhat enforce that we are passing a user we have in the DB. I agree that even the previous version did not enforce that completely but still.